hexrd/ui/calibration/wppf_options_dialog.py

```diff
@@ -420,6 +421,136 @@ def create_wppf_params_object(self):
             params.add(**kwargs)
         return params
 
+    def add_material_parameters(self):
```

Comment:
I think it'd be ideal if this were a function called by both `hexrd.WPPF` to add parameters, and this class as well. That way, if anything changes in the code, we won't have to change it in two places.

But we can leave it this way if you'd prefer.

Reply:
I think thats a good idea. It'll require some changes to the `WPPF` module in `hexrd`. I'll have a look at this.